### PR TITLE
Add basic playbooks validations

### PR DESCRIPTION
**Pull request by ityankel**
Add very initial validation for playbook resources - to validate that json file contains a valid ARM template schema, and simple validations on the content (if template contains a playbook resource, its name and location match the expected convention).
After completing this PR I'll add more complex validations on other parts of the template - API connections, variables, metadata, etc.

### CODE REVIEW

File: .script/playbooksValidator.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+let fileKinds = ["Modified"];
+let CheckOptions = {
+  onCheckFile: (filePath: string) => {
+    return IsValidTemplate(filePath);
+  },
+  onExecError: async (e: any, filePath: string) => {
+    console.log(`Playbooks validation failed. File path: ${filePath}. Error message: ${e.message}`);
+  },
+  onFinalFailed: async () => {
+    logger.logError("An error occurred, please open an issue");
```
Comment (Amitbergman) [inline .script/playbooksValidator.ts:48]: where will we see these logs?

Comment (Amitbergman) [reply .script/playbooksValidator.ts:48]: In the results of the test?

**Review by Amitbergman — APPROVED** [review]

File: .azure-pipelines/playbooksValidations.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+jobs:
```
Comment (Amitbergman) [inline .azure-pipelines/playbooksValidations.yaml:1]: How did you do it not required?
image

Comment (ityankel) [reply .azure-pipelines/playbooksValidations.yaml:1]: I have no idea :(

Comment (ityankel) [reply .azure-pipelines/playbooksValidations.yaml:1]: I'm fine with this not being required check, at least at the start, so we'll be able to verify these new validations work fine and don't block contribution. But I have no idea what controls it so I don't know how to change it to required later on

File: .script/playbooksValidator.ts (hunk 1 above)
Comment (ityankel) [reply .script/playbooksValidator.ts:48]: my understanding is that they appear in the logs when you open the build pipeline for the specific check

**Pull request merged**